api/trace/trace_context_propagator.go

```diff
@@ -42,6 +42,9 @@ type TraceContext struct{}
 var _ propagation.TextFormat = TraceContext{}
 var traceCtxRegExp = regexp.MustCompile("^[0-9a-f]{2}-[a-f0-9]{32}-[a-f0-9]{16}-[a-f0-9]{2}-?")
 
+// DefaultPropagator is the default TraceContext propagator
```

Comment:
I'd like to reword it a bit:
```suggestion
// DefaultPropagator is the default trace propagator.
```